src/MovieDetails.js

```diff
@@ -12,16 +12,16 @@ const MovieDetails = ({ movie, closeDetails }) => {
         <p>
           {movie.release_date} {movie.average_rating} {movie.rating}
         </p>
-        {/* {movie.genres.map((genre) => {
-        <div id={genre.id}>
-          <p>{genre}</p>
-        </div>;
-      })}
-      <h3>{movie.tagline}</h3>
-      <p>{movie.overview}</p>
-      <p>
-        {movie.budget} {movie.revenue}
-      </p> */}
+        {movie.genres.map((genre) => {
```

Comment:
I'd maybe pull this `map` out into its own variable (this is called memoizing), and then reference that variable in your return. This will help to keep the return and renders very clean and readable!